### DISCUSSION TITLE
entrance rando?

### DISCUSSION THREAD

**Discussion by Not-Chezit** [phst-randomizer/ph-randomizer#277] [Ideas]
https://github.com/AuriRex/zdspy

found this and have no clue how helpful or how possible it is to implement it.

**Comment by mike8699** [comment]
Thanks! Yes I have seen that library before. I haven't looked into the code yet, but I'm hoping it will be of some use for implementing entrance rando once we get to that point (at the moment the main focus is on getting a fully functional item randomizer working, but entrance rando is on the radar after that)

**Reply by Not-Chezit** [reply]
awesome! can't wait to see what you guys come up with.
